Error from server (AlreadyExists): pods "anhpham-project4-prediction" already exists
NAME                          READY   STATUS    RESTARTS   AGE
anhpham-project4-prediction   1/1     Running   0          16m
Forwarding from 127.0.0.1:8080 -> 80
Forwarding from [::1]:8080 -> 80
Handling connection for 8080
kubectl logs anhpham-project4-prediction
 * Serving Flask app "app" (lazy loading)
 * Environment: production
   WARNING: Do not use the development server in a production environment.
   Use a production WSGI server instead.
 * Debug mode: on
 * Running on http://0.0.0.0:80/ (Press CTRL+C to quit)
 * Restarting with stat
 * Debugger is active!
 * Debugger PIN: 310-243-920
[2023-04-17 16:23:39,773] INFO in app: JSON payload: 
{'CHAS': {'0': 0}, 'RM': {'0': 6.575}, 'TAX': {'0': 296.0}, 'PTRATIO': {'0': 15.3}, 'B': {'0': 396.9}, 'LSTAT': {'0': 4.98}}
[2023-04-17 16:23:39,786] INFO in app: Inference payload DataFrame:
   CHAS     RM    TAX  PTRATIO      B  LSTAT
0     0  6.575  296.0     15.3  396.9   4.98
[2023-04-17 16:23:39,792] INFO in app: Scaling Payload:
   CHAS     RM    TAX  PTRATIO      B  LSTAT
0     0  6.575  296.0     15.3  396.9   4.98
[2023-04-17 16:23:39,795] INFO in app: Output Prediction:
[20.35373177134412]
127.0.0.1 - - [17/Apr/2023 16:23:39] "POST /predict HTTP/1.1" 200 -